src/static/js/proposals/proposal_form.js

```diff
@@ -0,0 +1,18 @@
+(function ($) {
+
+    if ($('select#id_recording_policy').find(':selected').val() == 'True') {
+        $('#div_id_live_stream_policy').hide();
+        $('select#id_live_stream_policy').val('True');
+    }
+
+    $('select#id_recording_policy').change(function(e){
+        if ($(this).val() == 'False') {
+            $('#div_id_live_stream_policy').fadeIn();
+            $('select#id_live_stream_policy').val('False');
+        } else {
+            $('#div_id_live_stream_policy').fadeOut();
+            $('select#id_live_stream_policy').val('True');
+        }
+    });
+
+})(jQuery);
```

Comment:
nit: add a newline at the end of the file

Reply:
Resolved in this [commit](https://github.com/pycontw/pycon.tw/pull/983/commits/15e661a6295982954459d1c52e3005d611b79731)